ee/vellum_ee/workflows/display/nodes/base_node_vellum_display.py

```diff
@@ -40,6 +41,8 @@ def get_target_handle_id_by_source_node_id(self, source_node_id: UUID) -> UUID:
         return self.get_target_handle_id()
 
     def get_source_handle_id(self, port_displays: Dict[Port, PortDisplay]) -> UUID:
-        default_port = self._node.Ports.default
+        unadorned_node = get_unadorned_node(self._node)
+        default_port = unadorned_node.Ports.default
```

Comment:
we always want to point to the source handle id of the unadorned node. if we want to change this, we should move the port displays we care about from the prompt node display to the try node display annotation